prov/gni/include/gnix_progress.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2016 Cray Inc. All rights reserved.
+ * Copyright (c) 2016 Los Alamos National Security, LLC.
```

Comment:
This is a new file.  It should only have the Cray copyright for 2017 (no LANL copyright).